docs/DEVELOPMENT.md

```diff
@@ -2,30 +2,35 @@
 
 ## Prerequisites
 - Golang - 1.21.x
-- Operator SDK version - 1.28.0
 ```
 export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
 export OS=$(uname | awk '{print tolower($0)}')
-export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.28.0
-curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
-install -m 755 operator-sdk_linux_amd64 ${SOME_DIR_IN_YOUR_PATH}/operator-sdk
 ```
 - podman, podman-docker or docker
 - Access to OpenShift cluster (4.12+)
 - Container registry to storage images
+- operator-sdk
+- opm
+
+### Obtaining opm and operator-sdk
+Makefile downloads operatorsdk 1.28.0 and opm 1.28.0 into ./bin.
+```
+make opm
+make operator-sdk
+PATH=$PWD/bin:$PATH
+```
 
 ### Get a token on registry.ci.openshift.org
 Our builder and base images are curated images from OpenShift.
 They are pulled from registry.ci.openshift.org, which require an authentication.
 To get access to these images, you have to login and retrieve a token, following [these steps](https://docs.ci.openshift.org/docs/how-tos/use-registries-in-build-farm/#how-do-i-log-in-to-pull-images-that-require-authentication)
 
-In summary:
-- login to one of the clusters' console
-- use the console's shortcut to get the commandline login command
-- log in from the command line with the provided command
-- use "oc registry login" to save the token locally
-
-### Using public images
+#### Using the token
+Set the authfile used by docker-build if login was successful.  Ex:
+```
+export IMAGE_AUTH='--authfile=~/.docker/config.json'
```

Comment:
Is this really needed? Doesn't docker honour the default auth file (`$HOME/.docker/config.json`) automatically if present ?

Reply:
`docker-build` did not use the auth I had in `$HOME/.docker/config.json` until I added
 `--authfile=$HOME/.docker/config.json`

If the user has not set IMAGE_AUTH, the Makefile will leave it out